packages/guides/src/Nodes/Configuration/ConfigurationBlockNode.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace phpDocumentor\Guides\Nodes\Configuration;
+
+use phpDocumentor\Guides\Nodes\AbstractNode;
+
+/** @extends AbstractNode<list<ConfigurationTab>> */
+final class ConfigurationBlockNode extends AbstractNode
+{
+    /** @param list<ConfigurationTab> $tabs */
```

Comment:
I would give each ConfigurationBlock a unique identifier that can be used to reference the tab parent when rendering the tabs etc

Reply:
how would that look like? an id based on the hashes of all tabs?

Reply:
per document counter. So if you display more then one per page you can still give them unique ids in HTMl?

Reply:
I think we cna leave that for now